util/venv-dependencies.sh

```diff
@@ -59,12 +60,20 @@ if [ "$#" -ne 1 ]; then
    invocation_problems=1
 fi
 
-echo "Checking for pip2..."
+PIP="pip"
+if pip -V 2>/dev/null | grep -q 'python 3'; then
+    echo "
+
+On your system, 'pip' appears to invoke Python 3's pip. This might cause problems.
+This script will try switching to 'pip2', but your pip2 install might be broken too.
+
+"
+    PIP="pip2"
+fi
 
-if ! pip2 --version >/dev/null 2>&1; then
+if ! "$PIP" >/dev/null 2>&1; then
```

Comment:
Will running "$PIP" without args do the expected thing with this conditional? I thought it might return non-zero without args. Should it be "$PIP" --version?